src/search/cegar/transition_system.cc

```diff
@@ -326,11 +326,15 @@ int TransitionSystem::get_num_loops() const {
 
 void TransitionSystem::print_statistics(utils::LogProxy &log) const {
     if (log.is_at_least_normal()) {
+#ifndef NDEBUG
         int total_incoming_transitions = 0;
+#endif
         int total_outgoing_transitions = 0;
         int total_loops = 0;
         for (int state_id = 0; state_id < get_num_states(); ++state_id) {
+#ifndef NDEBUG
             total_incoming_transitions += incoming[state_id].size();
+#endif
             total_outgoing_transitions += outgoing[state_id].size();
             total_loops += loops[state_id].size();
         }
```

Comment:
We could use `utils::unused_variable(total_incoming_transitions);` instead of the preprocessor directives. Same below.